packages/x-svg-converter/src/svg-to-vue.ts

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import { exec } from 'child_process';
+import { CommandParameters, SVGInfo } from './types';
+import { applyXDSFormat } from './regex-utils';
+
+if (require.main === module) {
+  svgToVue();
+}
+
+/**
+ * Load SVGs from provided folder, turn them into Vue Components and apply
+ * prettier to them. By default, the source SVGs are deleted unless the `keepSVGs`
+ * parameter is passed.
+ *
+ */
+export function svgToVue(): void {
+  const svgList = loadSVGsFromFolder();
```

Comment:
Should we catch here the errors and give a more user-friendly output?
Like showing a text with the usage of the script or something.

Reply:
I'm guessing you want to catch the errors that we produce in `loadSVGsFromFolder`, `getParams` and `runPrettier`. In that case, don't see why we should catch them here because we can modify the message directly where we are throwing them (and that way we won't catch and format unexpected errors).

Regarding the friendlier output. The script is intended for users of the library, so I don't see the messages too difficult to understand, but if you have a proposal I can change them.

Reply:
I was referring to just show in the console the error message, instead of all the error stack.
A solution that comes to my mind is to create a custom error class, throw using this class, and in the try/catch check whether the error is a controlled error or not.
If it is a controlled error, showing only the error in the console should be enough, if it is not, it can re-throw the error.
I think that maybe this can be nice to have, but it is not a required behavior, so it is up to you.

Reply:
Still not convinced xD but we can meet to talk about it.